File: backend/src/main/java/com/logistics/domain/warehouse/domain/WorkStatus.java
package com.logistics.domain.warehouse.domain;

/** 피킹·포장 작업 상태. 포장 작업은 IN_PROGRESS 없이 WAITING → COMPLETED로 끝난다. */
public enum WorkStatus {
    WAITING,
    IN_PROGRESS,
    COMPLETED,
}

File: frontend/src/constants/workStatus.ts
import { WorkStatus } from '@/types/work';
import { Tone } from '@/types/ui';

export const WORK_STATUS_LABEL: Record<WorkStatus, string> = {
  WAITING: '대기',
  IN_PROGRESS: '진행중',
  COMPLETED: '완료',
};

export const WORK_STATUS_TONE: Record<WorkStatus, Tone> = {
  WAITING: 'gray',
  IN_PROGRESS: 'blue',
  COMPLETED: 'green',
};
